apps/frontend/src/app/3_organisms/ZeroLocForm/CurrentTroveData.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { FC, ReactNode, useMemo } from 'react';
+
+import { useTranslation } from 'react-i18next';
+
+import { HelperButton } from '@sovryn/ui';
+
+import { CRatioIndicator } from '../../2_molecules/LOCStatus/components/CRatioIndicator/CRatioIndicator';
+import { formatValue } from '../../../utils/math';
+
+export type CurrentTroveDataProps = {
+  creditValue: string;
+  collateralValue: string;
+  rbtcPrice: string;
+  className?: string;
+};
+
+export const CurrentTroveData: FC<CurrentTroveDataProps> = ({
+  className,
+  creditValue,
+  rbtcPrice,
+  collateralValue,
+}) => {
+  const { t } = useTranslation();
+  const collateralRatio = useMemo(
+    () =>
+      ((parseFloat(collateralValue) * parseFloat(rbtcPrice)) /
+        parseFloat(creditValue)) *
+      100,
+    [collateralValue, creditValue, rbtcPrice],
+  );
+
+  return (
+    <div className={className}>
+      <div className="w-full flex flex-row justify-between items-center mb-12 gap-4">
+        <Column
+          label={t('LOCStatus.currentDebt')}
+          value={`${formatValue(Number(creditValue), 2)} ZUSD`}
+        />
+        <Column
+          label={t('LOCStatus.currentCollateral')}
+          value={`${formatValue(Number(collateralValue), 4)} RBTC`}
+        />
```

Comment:
I don't like that `ZUSD` and `RBTC` are hardcoded like this. Create new constants (`debtToken` and `collateralToken`) and use them here and in other appropriate places.